src/Input.test.jsx

```diff
@@ -0,0 +1,81 @@
+import { render, fireEvent } from '@testing-library/react';
+
+import Input from './Input';
+
+// Input test
+// ToDo: "등록" 버튼 클릭 후 value 공란으로
+
+describe('Input test', () => {
+  const handleChange = jest.fn();
+  const handleClick = jest.fn();
+
+  const renderInput = (value) => render(
+    <Input
+      value={value}
+      onChange={handleChange}
+      onClick={handleClick}
+    />,
+  );
```

Comment:
```suggestion
  const renderInput = (value) => render((
    <Input
      value={value}
      onChange={handleChange}
      onClick={handleClick}
    />
  ));
```

괄호를 2중으로 감싸주면 콤마를 없앨 수 있어요.

Reply:
코멘트 감사드립니다. :)